docs/hugo/content/guide/metrics.md

```diff
@@ -7,19 +7,20 @@ The metrics exposed fall into two groups: Azure based metrics, and reconciler me
 
 ## Toggling the metrics
 
-By default, metrics for ASOv2 are turned on and can be toggled by the following options:
+By default, secure metrics for ASOv2 are turned on and can be toggled by the following options:
 
--  ### ASOv2 Helm Chart
+### ASOv2 Helm Chart
 
     While installing the Helm chart, we can turn the metrics _**on**_ and _**off**_ and set the metrics expose address using the 
     below settings. Also, we can change the settings inside `values.yaml` file for ASOv2 Helm chart.
 
     ```
     --set metrics.enable=true/false (default: true)
-    --set metrics.address=0.0.0.0:8080 (default)    
+    --set metrics.secure-metrics=true/false (default: true)
```

Comment:
minor: I think call this just `metrics.secure`? The fact it's metrics is sorta implied given the path? Just like we don't call it `metrics.enableMetrics`?